pkg/controller/crd/group_crd.go

```diff
@@ -60,10 +60,11 @@ type GroupSpec struct {
 
 // PodDetails information
 type PodDetails struct {
-	Name       string
-	StartTime  meta_v1.Time
-	EndTime    meta_v1.Time
-	Containers []*Container
+	Name          string
+	StartTime     meta_v1.Time
+	EndTime       meta_v1.Time
+	Containers    []*Container
+	PodVolumesMap map[string]*PersistentVolumeClaim
```

Comment:
PodVolumesMap >> rename to PodVolumes or something similar.
We don't need to specify the type in name.

Reply:
Done